packages/react-app/src/data/challenges.js

```diff
@@ -5,48 +5,56 @@ export const challengeInfo = {
     disabled: false,
     description: "Build a simple NFT example",
     previewImage: "https://user-images.githubusercontent.com/526558/124386983-48965300-dcb3-11eb-88a7-e88ad6307976.png",
+    badgeId: "0"
```

Comment:
Yes, since the contract's mint function takes a uint as a param I think having a JS number makes more sense.

Fixed on 10b03afff8edf4a5b60cfad4a6061355f5df49e6